src/components/Chip/Chip.js

```diff
@@ -1,15 +1,48 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { hightlightSubString } from "../../utils";
 
 import "./Chip.scss";
 
-const Chip = ({ value, lead = "", onDismiss, selected }) => {
+const Chip = ({
+  value,
+  lead = "",
+  onClick,
+  onDismiss,
+  selected,
+  subString = "",
+}) => {
+  // When user tabs over chip and presses Enter or Space key, chip will trigger
+  // click functionality
+  const onKeyPress = (e) => {
+    // charCode 32 = Space key
+    // charCode 13 = Enter key
+    if (e.charCode === 32 || e.charCode === 13) {
+      if (onClick instanceof Function) {
```

Comment:
`if (onClick)` is probably more idiomatic, any reason you need this?

Reply:
This was just to prevent a fatal error where someone might pass the onClick as an empty string or object.